### ISSUE TITLE
Constants on the stack are corrupted after opening nonexisting help page

### ISSUE THREAD

**Issue by mahi2003** [c3d/DB48X-on-DM42#953]
1. Put equations with constants `π` (pi) and `e` (Euler's number) on the stack:
```
'2+π'
'1+e'
```
2. Open a non existing help page (for example hold a key like [1/x]). Once the "no help for..." error message appears the stack has changed to:
```
'2+Math'
'1+π'
```
`π` was changed to `Math` and `e` replaced with `π`!

3. Press the down button to edit the item in stack level 1, but cancel with [EXIT]. The original stack is restored.

It is however not just a visualization issue. If you execute [→Num] after step 2, stack level 1 becomes 4.1415 (1+pi)... whereas it should have been 3.7182... (1+e).

**Comment by c3d** [comment]
This is a duplicate of #943, and is already fixed on the `dev` branch.

**Comment by c3d** [comment]
Original user report:

https://www.hpmuseum.org/forum/thread-20157-post-187618.html#pid187618

**Comment by mahi2003** [comment]
Confirmed fixed in 0.7.7!
